tools/src/launcher/launcher-config.yaml

```diff
@@ -29,6 +29,9 @@ nodeArgs:
   - "--configuration-key"
   - "mainnet_dryrun_full"
 
+# Path to directory where state is stored
+statePath: ./
```

Comment:
Note that this file seems to only be used for documentation. I believe we need to add this `statePath` in more places, like in the `scripts/launch/demo.sh`

Reply:
Most recent commit adds it to demo.sh. @KtorZ tested and confirmed that he could no longer attempt to launch multiple demo clusters after the change.